src/content/blog/posts/moj-de-tech-radar.md

```diff
@@ -0,0 +1,81 @@
+
+## Ok, so what is a tech radar?
+
+A tech radar is a tool that maps out the key technologies your team is using or considering, represented as “blips.” These blips are arranged into concentric rings - **Adopt, Assess, Retain, and Replace** - which reflect the status of each technology, indicating whether we plan to adopt, phase out, or maintain it and split into quadrants, representing the different kinds of blips - languages, tools, etc.
+
+Originally pioneered by [ThoughtWorks](https://www.thoughtworks.com/radar) and further refined by [Zalando](https://engineering.zalando.com/tags/tech-radar.html), the tech radar provides a visual framework - much like an archery target - for understanding our tech landscape. It helps align technology choices across teams and guides informed decisions about which technologies to continue using or discontinue.
+
+### Current Implementation
+
+Our implementation follows the approach outlined by ThoughtWorks, focusing on the following rings:
+
+- **ADOPT**: Proven technologies and techniques we trust to meet our needs at scale. These are widely used in production, low-risk, and recommended for broad adoption.
+- **ASSESS**: Promising technologies with clear potential. These are worth researching and prototyping, though they carry higher risks as they are new and untested in our organisation. Some engineers may already be experimenting with them.
+- **RETAIN**: Technologies we currently use and are satisfied with but do not plan to adopt in new projects.
+- **REPLACE**: Outdated technologies or techniques that have been superseded. We will stop using them for new projects and plan to phase them out of existing ones.
+
+And split into the following quadrants:
+- **Platforms**:
+- **Tools**: Key software and technologies. These include ruff and Docker, amongst others.
+- **Languages** and Frameworks: Programming languages and common frameworks used in Data Engineering, such as dbt, pyarrow or pyspark.
+- **Techniques**: Common techniques used in software development and Data/Analytical Engineering. These include agile, architectural design records (ADRs) and Kimball (for dimensional modelling).
+
+## Our Tech Radar journey
+
+We began our first test run of the tech radar as a simple Miro whiteboarding exercise, where the Data Engineering team came together to identify key technologies and techniques, assigning them to relevant categories. [Add a quick note on why we wanted to migrate away from Miro?] [Snapshot to the miro board]
+
+In 2020, Zalando open-sourced their adapted [tech radar](https://github.com/zalando/tech-radar), which we’ve since adopted. The setup requires only a central JSON file containing details of each blip, including the date, quadrant, and ring it belongs to. This data then automatically populates the radar.
```

Comment:
```suggestion
In 2020, Zalando open-sourced their JavaScript-based [tech radar](https://github.com/zalando/tech-radar), which we’ve since adopted. The setup requires only a central JSON file containing details of each blip, including the date, quadrant, and ring it belongs to. This data then automatically populates the radar which can be published to static website host such as GitHub pages.
```